.travis.yml

```diff
@@ -6,4 +6,5 @@ cache:
   directories:
     - node_modules
 script:
+  - ./node_modules/.bin/tsc packages/types/src/Metadata-md.ts; node packages/types/src/Metadata-md.js
```

Comment:
Would rather do this manually - currently the static version is generated like that. (So just change this to something inside package.json)